packages/web/components/Modal/SettleDealModal/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useForm } from 'react-hook-form';
+import { ReviewButton as Button } from '../CurateTechnologyModal/styles';
+import { toast } from '../../Toast';
+import { Modal, InfosContainer, Summary } from './styles';
+import { CurrencyInputField, InputField } from '../../Form';
+import { formatCurrencyToInt, formatMoney } from '../../../utils/helper';
+
+const SettleDealModal = ({ closeModal }) => {
+	const [totalValue, setTotalValue] = useState(0);
+	const form = useForm();
+	const { watch } = form;
+	const formValues = watch();
+
+	const onSubmit = () => {
+		toast.success('Pedido fechado com sucesso');
+		closeModal();
+	};
+
+	useEffect(() => {
+		const { quantity, unityPrice } = formValues;
+
+		if (quantity && unityPrice) {
+			const intCurrency = formatCurrencyToInt(unityPrice);
+			const total = intCurrency * quantity;
+
+			setTotalValue(total);
+		}
```

Comment:
There is a tiny "bug" here: if you type the amount and the unity price, it sums the total value correctly. However, if you delete one of the two values, the total amount displayed will be the last amount * unity price calculated before one of the files got empty. The reason why this is happening is that it dispatches the userEffect function, but it doesn't calculate the totalValue again, because of the `if` clause. Suggestion:
```
if(quantity && unityPrice){

} else {
   setTotavalValue(0)
}

Reply:
Thanks for pointing this, tired eyes makes that kind of mistake 😯